### ISSUE TITLE
dmw_package

### ISSUE THREAD

**Issue by overload08** [domogik/domoweb#110]
Add capability to install widget/icons set with command line like dmg_package

**Comment by vdomos** [comment]
good idea.

I also wondered whether the plugin should not provide the widget if specific.

**Comment by overload08** [comment]
Good idea too. Using dmg_package to install plugin and widget too. Maybe will be more simple

**Comment by Cereal2nd** [comment]
Hmm, but this is a bit against the idea of separating the gui and the core.
So i rather create some sort of domoweb packages for icons and widgets.

**Comment by fritz-smh** [comment]
As commented by ceral and discussed also on irc, "dedicated" widgets won't be included in plugins.

**Comment by fritz-smh** [comment]
I create a ticket to handle icons. This ticket is no dedicated for handling widgets packs

**Comment by overload08** [comment]
#125 ?
